tox.ini

```diff
@@ -4,20 +4,22 @@ envlist=
     py27-django{111},
     py35-django{111,20,21},
     py36-django{111,20,21},
+    py37-django{22},
 
 [testenv]
 changedir=
     oidc_provider
 deps =
     mock
     psycopg2
-    pytest==3.6.4
+    pytest==5.4.3
```

Comment:
I'm not sure why this was pinned initially, but all tests to do seem to pass with this version update. Version 6.0.1 is out now, and you can try updating to that and seeing if this still works. But it doesn't matter at all for the functionality of this project itself, so don't worry about it if it's too much of a distraction from your current work.